test/ffmpeg/ffmpeg_command_test.dart

```diff
@@ -113,6 +113,42 @@ void main() {
           ),
         );
       });
+
+      test("converts a group of images to a video", () {
+        final command = FfmpegCommand.simple(
+          inputs: [
+            // file%d.png represents images named like file0.png, file1.png, file2.png, etc.
+            FfmpegInput.asset("assets/images/file%d.png"),
+
+            FfmpegInput.asset("assets/audio/audio.mp3"),
+          ],
+          args: [
+            const CliArg(name: 'y'),
+            const CliArg(name: 'c:a', value: 'aac'),
+            const CliArg(name: 'qscale:v', value: '1'),
+          ],
+          outputFilepath: 'assets/output/file.mp4',
```

Comment:
Would you really name your output `file.mp4`?

Reply:
I'll update it to something more appropriate.